tests/PolyType.Tests/CollectionShapeTests.cs

```diff
@@ -68,7 +68,7 @@ public InternalDictionaryOfIntWithPublicConstructor()
         }
     }
 
-    [GenerateShape<PublicListOfIntWithInternalConstructor>]
+    [GenerateShapeFor<PublicListOfIntWithInternalConstructor>]
```

Comment:
Are there any markdown/docs files that also need updating?

Reply:
Yes, and done.